getting-started/what-you-need.md

```diff
@@ -1,102 +1,116 @@
+   * You can download it once from your [Telerik account](https://www.telerik.com/account/) and then use it without an Internet connection.
+   * It allows you to install the [VS extensions]({%slug getting-started-vs-integration-overview%}) and use various [templates for creating new projects]({%slug getting-started-vs-integration-new-project%}) and a [wizard for enabling existing projects to use Telerik components]({%slug getting-started-vs-integration-convert-project%}).
+   * It does not provide information about new versions.
+   * Depending on your setup, it may require elevated privileges to run the installation wizard.
+   * It provides an offline version of the [demos](https://demos.telerik.com/blazor-ui/).
 
-* The [Automated MSI installer]({%slug installation/msi%}) - You can download it from your [Telerik account](https://www.telerik.com/account/) and then use it without an Internet connection. It does not provide information about new versions. Depending on your setup, it may require elevated privileges to run the install wizard. It provides an offline version of the [demos](https://demos.telerik.com/blazor-ui/) and allows you to install the [VS extensions]({%slug getting-started-vs-integration-overview%}).
+* The [ZIP archive]({%slug installation/zip%}) package:
 
-* The [ZIP archive]({%slug installation/zip%}) package - You can download it from your [Telerik account](https://www.telerik.com/account/) and then use it without an Internet connection. It does not provide information about new versions and does not require installation. It also provides an offline version of the [demos](https://demos.telerik.com/blazor-ui/).
+   * You can download it once from your [Telerik account](https://www.telerik.com/account/) and then use it without an Internet connection.
+   * It does not provide information about new versions and does not require installation. 
+   * It provides an offline version of the [demos](https://demos.telerik.com/blazor-ui/).
 
-* The standalone `.nupkg` files - They are the bare minimum that is required. To use them, follow the instructions for using the [ZIP archive]({%slug installation/zip%}), but download the `.nupkg` files instead.
+* The standalone `.nupkg` files:
 
-@[template](/_contentTemplates/common/get-started.md#nuget-update-note)
+   * They are the bare minimum that is required. 
+   * To use them, follow the instructions for using the [ZIP archive]({%slug installation/zip%}), but download the `.nupkg` files instead.
 
+## Adding the Client Assets
 
-## Client Assets
+For the Telerik UI for Blazor components to look and behave as expected in the browser, add the [Telerik stylesheet](#telerik-stylesheet) and the [JS Interop file](#telerik-js-interop-file) to your main index file. Depending on the Blazor hosting model and framework version, your project's main index file will differ:
```

Comment:
```suggestion
The Telerik UI for Blazor components require a [Telerik stylesheet](#telerik-stylesheet) and a [JS Interop file](#telerik-js-interop-file) in the app's main index file. Depending on the Blazor hosting model and framework version, this index file will differ:
```

Reply:
Well, it is very basic knowledge that stylesheets are related to looks and JavaScript files are related to interactivity. And both of them are needed in the browser. So I believe we don't have to explain this here at the cost of longer text that is harder to read.